### DISCUSSION TITLE
[Question]: Use standard quality, instead of hd for DALLE3

### DISCUSSION THREAD

**Discussion by dannykorpan** [danny-avila/LibreChat#1632] [Q&A] [answered]
### What is your question?

How is is possible to force LibreChat to use the parameter quality: standard, instead of hd for DALLE3?

### More Details

example

REQUEST TO DALL-E-3
{
  "prompt": "A comic strip featuring an IT administrator as a superhero in a company called 'youknow' doing weird things in a server room. The superhero IT administrator has a cape and a mask, and is wearing a t-shirt with the 'youknow' logo. In the first panel, the superhero is seen juggling servers while balancing on a stack of hard drives. In the second panel, the superhero is using a keyboard as a shield to block a laser beam coming from a malfunctioning server. In the third panel, the superhero is using a cable as a lasso to capture a rogue robot that is causing chaos in the server room. In the final panel, the superhero is sitting on a server rack, surrounded by a pile of pizza boxes, with a satisfied smile on their face. The comic strip has a vibrant and colorful art style.",
  "style": "vivid",
  "quality": "hd",
  "size": "1024x1024"
}

### Screenshots

<img width="723" alt="grafik" src="https://github.com/danny-avila/LibreChat/assets/8004507/51e834ae-22c5-4529-b94c-e8e617eb8c92">

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment, marked as answer]
Currently you can’t without editing the tool code. You can CTRL+F “DALLE3” in the project and adjust it there.

I do plan on being able to “prime” tool calls with predefined inputs from the frontend in an upcoming update.

**Reply by danny-avila** [reply]
You can also change the system prompt to coerce the LLM to prefer “standard”.

https://docs.librechat.ai/install/configuration/dotenv.html

**Reply by dannykorpan** [reply]
Thank you, I will try it!
